sklearn/svm/tests/test_svm.py

```diff
@@ -985,6 +985,31 @@ def test_ovr_decision_function():
     assert np.all(pred_class_deci_val[:, 0] < pred_class_deci_val[:, 1])
 
 
+@pytest.mark.parametrize("svc", [svm.SVC, svm.NuSVC])
+def test_svc_ovr_tie_breaking(svc):
+    """Test if predict breaks ties in OVR mode.
+    Related issue: https://github.com/scikit-learn/scikit-learn/issues/8277
+    """
+    X, y = make_blobs(random_state=27)
+
+    np.random.seed(0)
```

Comment:
That would affect other tests wouldn't it? Better to pass random_state to SVC.

Reply:
+1